src/model/ClientModel.java

```diff
@@ -36,12 +36,12 @@ public Player getPlayer(){
 	}
 	
 	public void addMessage(String m){
-		messageCount++;
-		if (messageCount == 10) {
+		//messageCount++;
```

Comment:
Yes, I just wanted to leave it there in case we still wanted to limit the number of messages being shown in some way.